README.md

```diff
@@ -106,6 +102,8 @@ After checking out the repo, run `bin/setup` to install dependencies. Then, run
 
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
+In troubleshooting situtations, it may be convenient to cloning the repository locally and reference that location from the vets-api Gemfile. This makes it easy to use `byebug` etc to debug the repo.
```

Comment:
This can be done with any gem at any time. Calling it out explicitly seems redundant and outlines [our need for a way to enable debug logging](https://github.com/department-of-veterans-affairs/betamocks/issues/12)